R/utils.R

```diff
@@ -146,17 +146,15 @@ error_if_less_than <- function(using_version, tested_version) {
   comp <- compare_connect_version(using_version, tested_version)
   if (is.na(comp)) {
     msg <- glue::glue(
-      "WARNING: This API requires Posit Connect version {tested_version} ",
+      "WARNING: This feature requires Posit Connect version {tested_version} ",
       "but the server version is not exposed by this Posit Connect instance. ",
-      "You may be experience errors when using this functionality."
+      "You may experience errors when using this functionality."
     )
     warn_once(msg)
   } else if (comp < 0) {
     msg <- glue::glue(
-      "ERROR: This API requires Posit Connect version {tested_version} ",
-      "but you are using {using_version}. Please use a previous version ",
-      "of the `connectapi` package, upgrade Posit Connect, or review the API ",
-      "documentation corresponding to your version."
```

Comment:
I changed the error messages.

- The phrase "this API requires…" doesn't really make sense. Users of the package will be thinking in terms of features or functions.
- The guidance in the error case was confusing. Using a previous version of `connectapi` would not help! If the feature doesn't exist on an old version of Connect, using an older `connectapi` will… just remove the `connectapi` references to that feature. Reviewing the API docs won't help either, because users won't see this error if they are directly calling the API, only if they're using existing functions.